guide/14-deep-learning/How CycleGAN Works.ipynb

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
<p>could start by describing what type of model is cyclegan or about image-to-image translation, then can continue "There are situations.......".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>